packages/react-scripts/config/webpack.config.js

```diff
@@ -140,8 +141,17 @@ module.exports = function (webpackEnv) {
   // common function to get style loaders
   const getStyleLoaders = (cssOptions, preProcessor) => {
     const loaders = [
-      isEnvDevelopment && require.resolve('style-loader'),
-      isEnvProduction && {
+      (isEnvDevelopment || process.env.REACT_APP_INJECT_STYLES) && {
```

Comment:
We need to check against the string 'true' because if someone sets it to the string 'false', it would return as truthy
```suggestion
      (isEnvDevelopment || process.env.REACT_APP_INJECT_STYLES === 'true') && {
```